### DISCUSSION TITLE
DeepXDE package - Cores

### DISCUSSION THREAD

**Discussion by KostasVog20** [lululxvi/deepxde#1673] [Q&A]
Dear colleagues and professor @lululxvi 

I am currently utilizing the DeepXDE package on my computer and have noticed that it currently operates with 32 cores. However, I have access to a system with 64 cores and am interested in leveraging this additional computational capacity. Could you please advise me on how to configure the DeepXDE package to utilize the full extent of these cores?

Thank you for your assistance.

Best regards,

**Comment by lululxvi** [comment]
DeepXDE uses TensorFlow/PyTorch/JAX, so it is the problem of TensorFlow/PyTorch/JAX.

**Reply by KostasVog20** [reply]
Thank you  professor @lululxvi. I use TensorFlow to run DeepXDE. Can I do something to increase the number of available CPUs?
